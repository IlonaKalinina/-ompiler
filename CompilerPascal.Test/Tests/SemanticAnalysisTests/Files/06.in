program Test_06;
type
people = record
age : integer;
age : string;
end;
var
people : people;
begin
end.