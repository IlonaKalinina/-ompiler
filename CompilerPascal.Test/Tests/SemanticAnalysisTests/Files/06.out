(5,1) ERROR: Duplicate identifier "age"
